WordPress/Classes/Utility/Logging/CustomLogFormatter.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import CocoaLumberjack
+
+class CustomLogFormatter: NSObject, DDLogFormatter {
+
+    let logTimeStampFormatter: DateFormatter = {
```

Comment:
now this is sweet. Maybe we could change this to be a _shorter_ time stamp in the future, too?

As a separate PR I mean, but the potential to control that aspect is appealing 😀

Reply:
Oh the power we have! 😉